pyqtgraph/widgets/PenSelectorDialogbox.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'PenSelectorDialogbox.ui'
+#
+# Created by: PyQt4 UI code generator 4.11.4
+#
+# WARNING! All changes made in this file will be lost!
+
+from PyQt4 import QtCore, QtGui
```

Comment:
Do the import like the rest of pyqtgraph does it (and use nonspecific Qt), imports at tops
```suggestion
from ..Qt import QtCore, QtGui
from .ColorButton import ColorButton
```